### PR TITLE
[MINOR] Migrate 0.5.0-incubating-SNAPSHOT to 0.5.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by haoch**
Migrate `0.5.0-incubating-SNAPSHOT` to `0.5.0-SNAPSHOT` as Eagle graduated from incubator and becomes TLP

### CODE REVIEW

**Comment by anyway1021** [comment]
LGTM

**Comment by jhsenjaliya** [comment]
Hi Hao,
Search for "incubating" in entire project gave me 100+ files ( 98 xml files with some 5 test files and other readme's ), wondering if this PR covers all of them.

**Comment by anyway1021** [comment]
@haoch Hao, what's the plan of migration? Step by step, or complete it with 1 shot?

**Comment by haoch** [comment]
@jhsenjaliya good catch, will fix it accordingly. 
@anyway1021, this PR will only fix the version-related things in source code, please feel free to clean other steps if any missing in separated PR.

**Comment by haoch** [comment]
@jhsenjaliya I think i have resolved all "incubating" things, please help review again.

File: eagle-core/eagle-alert-parent/eagle-alert-app/pom.xml [hunk 1]
```diff
@@ -15,13 +15,12 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (jhsenjaliya) [inline eagle-core/eagle-alert-parent/eagle-alert-app/pom.xml]: got it, Thanks

**Comment by jhsenjaliya** [comment]
@haoch, found 1 last reference in LICENSE file having text like "Apache Eagle (incubating) Subcomponents:" , otherwise looks good.

**Comment by haoch** [comment]
@jhsenjaliya fixed as well. If no more problem, I will merge the PR.

**Comment by haoch** [comment]
Thanks @anyway1021 and @jhsenjaliya for reviewing. The pr has been merged now.

**Pull request closed without merging**